packages/react-components/src/components/UnsupportedBrowser.tsx

```diff
@@ -43,38 +33,15 @@ export interface UnsupportedBrowserProps {
   strings: UnsupportedBrowserStrings;
 }
 
-/* @conditional-compile-remove(unsupported-browser) */
-const UnsupportedBrowserContainer = (props: UnsupportedBrowserProps): JSX.Element => {
-  const { onTroubleshootingClick, strings } = props;
-  return (
-    <Stack styles={containerStyles}>
-      <Icon styles={iconStyles} iconName="UnsupportedBrowserWarning" data-ui-id="unsupportedBrowserIcon"></Icon>
-      <Text styles={mainTextStyles}>{strings.primaryText}</Text>
-      <Text styles={secondaryTextStyles}>{strings.secondaryText}</Text>
-      {onTroubleshootingClick && (
-        <Link
-          styles={linkTextStyles}
-          onClick={() => {
-            onTroubleshootingClick();
-          }}
-          data-ui-id="unsupportedBrowserLink"
-        >
-          {strings.moreHelpLink}
-        </Link>
-      )}
-    </Stack>
-  );
-};
-
 /**
  * UI to display to the user that the browser they are using is not supported by calling application.
```

Comment:
nit: calling is not an "application".

Perhaps say "not supported by the Azure Communication Services calling service"?